### PR TITLE
Bom 1020

### PR DESCRIPTION
**Pull request by Ayub-Khan**
**Description:** Describe in a couple of sentences what this PR adds

**JIRA:** Link to JIRA ticket

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green
- [x] Version bumped
- [x] Changelog record added
- [x] Documentation updated (not only docstrings)
- [x] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -52,13 +22,15 @@ whitelist_externals =
     rm
 deps =
     -r{toxinidir}/requirements/doc.txt
+    twine
```
Comment (awais786) [inline tox.ini]: Add this in quality.in

**Comment by jmbowman** [comment]
@Ayub-Khan Is anything currently blocking you on this?  I'd been holding off on reviewing it until tests started passing.

**Comment by Ayub-Khan** [comment]
@jmbowman some how i was not able to find the root cause why tests are failing against django22 on travis, they are passing locally. thn i started working on other issues and put this on hold.

**Comment by jmbowman** [comment]
I can reproduce the failures locally on your branch via `tox -e py35-django22`, unfortunately no insight yet into the underlying cause.

**Comment by Ayub-Khan** [comment]
@jmbowman this is ready for review.

File: tox.ini (hunk 1 above)
Comment (jmbowman) [reply tox.ini]: Agreed, `twine` should be added to the appropriate requirements file.

**Pull request merged**